doc/api/async_hooks.md

```diff
@@ -29,7 +29,7 @@ Following is a simple overview of the public API.
 const async_hooks = require('async_hooks');
 
 // Return the ID of the current execution context.
-const cid = async_hooks.currentId();
+const cid = async_hooks.currentAsyncId();
```

Comment:
The most technically correct name for `currentId()` would be `asyncIdOfCurrentExecutionScope()`.

you're right in that `triggerId` is an `asyncId`. it's the `asyncId` of the resource that is responsible for the callback being called. e.g.
```js
const async_hooks = require('async_hooks');
const net = require('net');
const print = process._rawDebug;

const server = net.createServer(connection => {
  print(connection._handle.getAsyncId() === async_hooks.triggerId());
  print(server._handle.getAsyncId() === async_hooks.currentId());
  server.close();
}).listen(8080, () => net.connect(8080).end());
```
(unfortunately there's a bug right now which doesn't properly set the `triggerId` for the `'connection'` callback)

The `currentId` is the `server`'s `asyncId` because the server is responsible for the current execution context. The `triggerId` is set to the `connection`'s `asyncId` because the `'connection'` callback would never been called if the new connection hadn't been made.